### PR TITLE
Add health check endpoint and tests

### PR DESCRIPTION
**Pull request by Devashish08**
### Issue Description
healthcheck endpoint for discord service. its going to be a get request

### Expected Behavior
---

### Current Behavior
none

### Screenshots
---

### Reproducibility

 - [ ] This issue is reproducible
 - [x] This issue is not reproducible

### Steps to Reproduce
1. 
2. 

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [ ] Medium
 - [x] Low

### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

### CODE REVIEW

File: routes/baseRoute.go [hunk 1]
```diff
@@ -8,4 +8,7 @@ import (
 
 func SetupBaseRoutes(router *httprouter.Router) {
 	router.POST("/", middleware.VerifyCommand(controllers.HomeHandler))
+
+	// Add the health check route
```
Comment (joyguptaa) [inline routes/baseRoute.go:12]: remove this comment

File: controllers/healthCheckHandler.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import (
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+func HealthCheckHandler(response http.ResponseWriter, request *http.Request, params httprouter.Params) {
+	response.Header().Set("Content-Type", "application/json")
+	response.WriteHeader(http.StatusOK)
+	response.Write([]byte(`{"status": "ok"}`))
```
Comment (joyguptaa) [inline controllers/healthCheckHandler.go:12]: @iamitprakash do we want to have a timestamp in the response as well?

**Comment by joyguptaa** [comment]
Add test coverage as well

**Comment by Devashish08** [comment]
> Add test coverage as well

i had already write tests for it

File: controllers/healthCheckHandler.go [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+func HealthCheckHandler(response http.ResponseWriter, request *http.Request, params httprouter.Params) {
```
[the remark this answers is not in the document]
Comment (Devashish08) [reply controllers/healthCheckHandler.go:16]: But now there is not any database integration in service and also didn't know about it should i add it also ?

[the remark this answers is not in the document]
Comment (yesyash) [reply controllers/healthCheckHandler.go:16]: - thanks for point this out closing this

**Review by joyguptaa — APPROVED** [review]

**Pull request merged**